src/platform/common/platform/fileSystem.node.ts

```diff
@@ -90,28 +89,28 @@ export class FileSystem extends FileSystemBase implements IFileSystemNode {
     }
 
     public async createLocalDirectory(path: string): Promise<void> {
-        await this.createDirectory(vscode.Uri.file(path));
+        await fs.ensureDir(path);
     }
 
-    async copyLocal(source: string, destination: string): Promise<void> {
-        const srcUri = vscode.Uri.file(source);
-        const dstUri = vscode.Uri.file(destination);
-        await this.vscfs.copy(srcUri, dstUri, { overwrite: true });
+    async copyLocal(
+        source: string,
+        destination: string,
+        options: { overwrite: boolean } = { overwrite: true }
+    ): Promise<void> {
+        await fs.copy(source, destination, { overwrite: options?.overwrite });
     }
 
     async readLocalData(filename: string): Promise<Buffer> {
-        const uri = vscode.Uri.file(filename);
-        const data = await this.vscfs.readFile(uri);
-        return Buffer.from(data);
+        return fs.readFile(filename);
```

Comment:
Using VSC API here is totally unnecessary its very slow. 
Since these are local only (the methods are `local` , hence easier & more efficient to just use native node modules.